packages/lesswrong/components/posts/LWPostsItem.tsx

```diff
@@ -475,7 +477,7 @@ const LWPostsItem = ({classes, ...props}: PostsList2Props) => {
             </PostsItem2MetaInfo>}
 
             {!!post.unreadDebateResponseCount && <PostsItem2MetaInfo className={classes.unreadDebateResponseContainer}>
-              <div className={classes.unreadDebateResponseCount} onClick={toggleComments}>
+              <div className={classes.unreadDebateResponseCount} onClick={toggleDialogueMessages}>
```

Comment:
Is this changing existing functionality for the older version of dialogues?

Reply:
I'll switch on `collabEditorDialogue`

Reply:
@rjmk Just to double-check - it seems like this removes the existing functionality to view recent comments for dialogues and replaces it entirely with functionality to view recent dialogue messages?

Reply:
No, that's not right. This is just for the unreadDebateResponseContainer, which is a little green debate icon that appears next to the comment counter

Reply:
Comments are down on line 498